src/script/cpp_api/s_security.cpp

```diff
@@ -257,6 +259,20 @@ bool ScriptApiSecurity::isSecure(lua_State *L)
 
 bool ScriptApiSecurity::safeLoadFile(lua_State *L, const char *path)
 {
+	lua_rawgeti(L, LUA_REGISTRYINDEX, CUSTOM_RIDX_SCRIPTAPI);
+	ScriptApiBase *script = (ScriptApiBase *) lua_touserdata(L, -1);
+	lua_pop(L, 1);
+	
+	Client *client = script->getClient();
```

Comment:
Are you asking to have this moved into the `if` statement below? The condition for the statement is dependant on this result.